tasks/secure-installation.yml

```diff
@@ -61,7 +73,7 @@
   with_items: "{{ mysql_root_hosts.stdout_lines|default([]) }}"
   when: >
     ((mysql_install_packages | bool) or mysql_root_password_update)
-    and ('5.7.' not in mysql_cli_version.stdout and '8.0.' not in mysql_cli_version.stdout)
+    and ('5.7.' not in mysql_cli_version.stdout and '8.' not in mysql_cli_version.stdout)
```

Comment:
The only downside to this change is maybe the comparison would break if there were a version like `5.8.0`, that version would be identified as `8.`... trying to think of a better way to narrow it. Maybe add in something to trim off the first two digits, and compare that to `8.`?

Reply:
I was thinking about this solution, but as far as I know, version 5.8 doesn't exist. However, everything works fine on version >= 5.7. If you want, I can change the version definition using trim.

Reply:
I think using trim would be ever so slightly more future proof. It feels more correct to me.